checker-qual/src/main/java/org/checkerframework/checker/immutability/qual/Assignable.java

```diff
@@ -0,0 +1,15 @@
+package org.checkerframework.checker.immutability.qual;
+
+import org.checkerframework.checker.initialization.qual.HoldsForDefaultValue;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.FIELD})
+@HoldsForDefaultValue
+public @interface Assignable {}
```

Comment:
Please add javadoc for everything (again, in the PICO repo).

Reply:
Sure, I will create a new PR to add the Javadoc.